client/src/components/common/applicant-info-1.vue

```diff
@@ -568,10 +597,24 @@ export default class ApplicantInfo1 extends Vue {
       (this.$refs.step1 as any).validate()
     }
   }
+
+  @Watch('isValid')
+  onValidChanged (val: boolean) {
+    if (val) {
+      this.$nextTick(() => {
+        // add classname to button text (for more detail in Sentry breadcrumbs)
+        this.$el.querySelector("#submit-back-btn-true > span")?.classList.add("applicant-back-btn")
+        this.$el.querySelector("#submit-continue-btn-true > span")?.classList.add("applicant-continue-btn")
+      })
```

Comment:
This adds a classname to the v-button span element, which is displayed in the Sentry breadcrumbs. I tried to use somewhat unique and helpful names to help with trouble-shooting.
You'll see similar code to this in several components that I updated.